crux-core/src/crux/kv/index_store.clj

```diff
@@ -698,6 +698,24 @@
       {:bytes-indexed (->> content-idx-kvs (transduce (comp (mapcat seq) (map mem/capacity)) +))
        :indexed-docs docs}))
 
+  (exclusive-avs [this eids]
+    (with-open [snapshot (kv/new-snapshot kv-store)
```

Comment:
can `unindex-eids` use this too?

Reply:
potentially - if we commit to this approach of eviction. Could also be done in a separate PR even then.

Reply:
yes raise issue to capture unindex-eids using